app/js/components/manageApps/ManageApps.jsx

```diff
@@ -699,7 +699,8 @@ export default class ManageApps extends React.Component {
       });
     }).catch(
       (error) => {
-        toastr.error(error);
+        error.message === "Network Error" ? toastr.error("There is no internet connection") : toastr.error(error.message);
+        this.handleApplist();
```

Comment:
Just an observation, you are toasting `toastr.error(error.message)` 
Are we certain that all other errors message which is not `Network Error` will be nicely formatted for the end user?

Reply:
@Charpell Yeah any other error message that is not Network error will be correctly formatted to the user as illustrated by the ternary expression.I have tested this